packages/base-driver/package.json

```diff
@@ -53,9 +53,12 @@
     "validate.js": "^0.13.0",
     "webdriverio": "^6.0.2",
     "ws": "^7.0.0"
-  },  
+  },
   "devDependencies": {
     "@appium/gulp-plugins": "^5.4.0",
     "@appium/eslint-config-appium": "^4.7.0"
+  },
+  "publishConfig": {
+    "access": "public"
```

Comment:
the 2.0 branch of basedriver (which this is based on) should probably only be published as "beta" until we explicitly opt out of that mode.

Reply:
ok

Reply:
wait--do you mean `next`?